.gitignore

```diff
@@ -1,2 +1,53 @@
 *.swp
 *.swo
+
+*.gem
```

Comment:
Could use some help on this `.gitignore` as I'm not familiar with everything that's here.

Reply:
Everything that's here is probably fine. As long as it doesn't exclude files we actually have or want to add.